editoast/src/schema/buffer_stop.rs

```diff
@@ -43,9 +43,11 @@ impl BufferStop {
             })
             .collect::<Vec<_>>();
 
-        diesel::insert_into(osrd_infra_bufferstopmodel)
-            .values(datas)
-            .execute(conn)?;
+        for data_chunk in datas.chunks(65534) {
```

Comment:
- Why your chunk size equals $2^{16} - 2$ and not $2^{16} - 1$?
- You could have a const variable that contains the value `2_usize.pow(16) - 1`

Reply:
Done and done :)